domains/abiapp789.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "abiapp789",
+    "email": "abiapp2024@gmail.com"
+  },
+  "record": {
+    "CNAME": "abiapp789.github.io"
```

Comment:
You should complete your website fiest, as of now i only see a "hello, welcome to my website" text with no css